site/docs/reference/authentication.md

```diff
@@ -0,0 +1,85 @@
+---
+sidebar_position: 1
+---
+
+# Authenticating Flow
+
+Read, write, and admin capabilities over Flow catalogs and the [collections](../concepts/collections.md) that comprise them
+are granted to Flow users through **roles**.
+
+Roles are granted in terms of **prefixes** within the Flow [namespace](../concepts/README.md#namespace).
+By default, each organization, or **tenant**, has a unique high-level prefix.
+For example, if you worked for Acme Co, your assigned tenant prefix would be `acmeCo/`.
+You may further divide your namespace however you'd like; for example `acmeCo/anvils` and `acmeCo/roadrunners`.
+When you name a collection, you can customize the prefix, and roles can be configured at any prefix level.
+This allows you to flexibly control access to your Flow data.
+
+The available roles are:
+
+* **`read`**: Allows the subject to read data from collections of the given prefix.
+
+* **`write`**: Allows the subject to read and write data from collections of the given prefix.
+
+* **`admin`**: Allows the subject to read and write data from collections of the given prefix,
+and to manage storage mappings, catalog specifications, and role grants within the prefix.
+The admin role also inherits roles granted to the prefix, as discussed below.
+
+## Subjects, objects, and inherited roles
+
+The entity to which you grant a role is called the **subject**, and the entity over which access is granted is called the **object**.
+The subject can be either a user or a prefix, and the object is always a prefix. This allows subjects to inherit nested roles,
+so long as they are granted the `admin` role.
+
+For example, user X of Acme Co has admin access to the `acmeCo/` prefix, and user Y has write access.
+A third party has granted `acmeCo/` read access to shared data at `outside-org/acmeCo-share/`.
+User X automatically inherits read access to `outside-org/acmeCo-share/`, but user Y does not.
+
+## Default authentication settings
+
+When you first sign up to use Flow, your organization is provisioned a tenant prefix, and your username is granted admin access to the prefix.
+Your prefix is granted write access to itself and read access to its logs, which are stored in the global `ops/` prefix.
+
+Using the same example, say user X signs up on behalf of their company, AcmeCo. User X is automatically granted `admin` access to the `acmeCo/` prefix.
+`acmeCo/`, in turn, has write access to `acmeCo/` and read access to `ops/acmeCo/`.
+
+As more users and prefixes are added, admins can [provision roles](#provisioning-roles) using the CLI.
+
+## Authenticating Flow in the web app
+
+You must sign in to begin a new session using the Flow web application.
+For the duration of the session, you'll be able to perform actions depending on the roles granted to the user profile.
+
+You can view the roles currently provisioned in your organization on the **Admin** tab.
+
+## Authenticating Flow using the CLI
```

Comment:
Just noting that "Authentication" _is_ the correct term for this section.

Also, it's maybe worth putting some kind of warning here? This workflow is hopefully very temporary, and will likely be updated before the CLI gets released more broadly.